TABLE.md

```diff
@@ -0,0 +1,25 @@
+# Built-in Decorators Synopsis
+
+|                           |                              `@wrap(f)`                                   |                                                       `@register(f)`                                                         |             `@expose(f)`                                                                                                     |             `@initialize(f)`          |
+|---------------------------|---------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------    |------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
+| **when**                  | while setting up the class                                                | after the class is created                                                                                                   | after the class is created                                                                                                   | after executing the class initializer |
+| **function f**            |                                                                           |                                                                                                                              |                                                                                                                              |                                       |
+| *parameters*              | - target: method (for method and accessor)<br>- target: class (for class) | - target: class (for static fields, static method and class)<br/>- target: class prototype (for instance fields and methods) | - target: class (for static fields, static method and class)<br/>- target: class prototype (for instance fields and methods) | - target: class instance              |
+|                           |                                                                           | - key (for public fields, methods or accessors)<br>- undefined (for private and class)                                       | - key: private name string                                                                                                   | - value: field value                  |
```

Comment:
It looks like some of these use a table row per parameter, and others use a single table row for several parameters.